modules/5_install/variables.tf

```diff
@@ -34,7 +34,7 @@ variable "region" {}
 variable "zone" {}
 variable "name_prefix" {}
 variable "node_prefix" {}
-
+variable "fips_compliant" {}
```

Comment:
don't remove the newline instead add it on a newline (maintain existing formatting for better distinction of diff options)